Package.swift

```diff
@@ -0,0 +1,77 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let headersSearchPath: [CSetting] = [.headerSearchPath("."),
+                                     .headerSearchPath("Base"),
+                                     .headerSearchPath("Debug"),
+                                     .headerSearchPath("Details"),
+                                     .headerSearchPath("Details/Transactions"),
+                                     .headerSearchPath("Layout"),
+                                     .headerSearchPath("Private"),
+                                     .headerSearchPath("Private/Layout"),
+                                     .headerSearchPath("TextExperiment/Component"),
+                                     .headerSearchPath("TextExperiment/String"),
+                                     .headerSearchPath("TextExperiment/Utility"),
+                                     .headerSearchPath("TextKit"),
+                                     .headerSearchPath("tvOS"),]
+
+let sharedDefines: [CSetting] = [
+                                // Disable "old" textnode by default for SPM
+                                .define("AS_ENABLE_TEXTNODE", to: "0"),
```

Comment:
Why is this disabled?
It seems to be enabled in `ASAvailability.h` does this not cause a mismatch between the Project version and the SPM version?

Reply:
Thanks for mention that, I guess the main reason was - I need that to be a default since `ASTextNode2` is a future I just enabled it by default :)

Reply:
```suggestion
                                .define("AS_ENABLE_TEXTNODE", to: "1"),
```

This option leads `undefined symbol: _OBJC_CLASS_$_ ASTEXTNODE2-` error when using `ASTextNode2`. If you can check, or if someone else meets this error, try changing it from 0 to 1.

Reply:
This option makes usage of ASTextNode2 a default. So in your code, you should reference it just like `ASTextNode`